dashboard/app/models/user.rb

```diff
@@ -2877,6 +2877,8 @@ def self.us_state_dropdown_options
       errors.add(:us_state, :blank)
```

Comment:
I think this is going to result in all the users who managed to create a new account without setting a US state being considered invalid. Whenever they go to update something on their user model, like their display name, validation will fail because US State is blank. You can verify this by creating a unit test where `country_code: 'US'` and `us_state: nil` then call `user.validate!`. We should try to have unit tests for all "states" our users are in. Let's pair to brainstorm a list of all of them and then write the tests together.